docs/docs/legal/terms-of-service-dedicated.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Dedicated Instance Terms
+custom_edit_url: null
+--- 
+## General
+
+Last revised: July 20, 2021
+
+### Background
+
+Within the scope of the Framework Agreement, the Customer may choose to purchase a subscription that requires a dedicated instance of ZITADEL. These additional terms for dedicated instance ("**Dedicated Instance Terms**") apply in addition to the Framework Agreement.
+
+### Service
+
+CAOS operates and manages a **Dedicated Instance** of ZITADEL in a private infrastructure environment dedicated for Customer's use and provides support services for the customer according the Booking, these terms, agreed [**Service Level Description**](service-level-description), and [**Support Service Descriptions**](support-services).
+
+Each Dedicated Instance consists, except agreed otherwise in writing, a multi-zonal high-availability configuration that guarantees loads up to the specified [rate limits](https://docs.zitadel.ch/docs/legal/rate-limit-policy#what-rate-limits-do-apply).
```

Comment:
```suggestion
Each Dedicated Instance consists, except agreed otherwise in writing, of a multi-zonal high-availability configuration that guarantees loads up to the specified [rate limits](rate-limit-policy#what-rate-limits-do-apply).
```